packages/clerk-js/src/core/fraudProtection.ts

```diff
@@ -47,10 +58,18 @@ export class FraudProtection {
       let resolve: any;
       this.inflightException = new Promise<unknown>(r => (resolve = r));
 
-      const captchaParams = await this.managedChallenge(clerk);
-
       try {
-        await this.client.getOrCreateInstance().sendCaptchaToken(captchaParams);
+        const captchaParams: any = await this.managedChallenge(clerk);
+        if (captchaParams?.captchaError !== 'modal_component_not_ready') {
+          await this.client.getOrCreateInstance().sendCaptchaToken(captchaParams);
+          this.captchaRetryCount = 0; // Reset the retry count on success
+        }
```

Comment:
If it's the first `/client` request on initialization it will always fail because we always initialize the components after the first `/client` request.